tests/fastai/test_fastai_autolog.py

```diff
@@ -0,0 +1,258 @@
+import pytest
+import numpy as np
+from tests.projects.utils import tracking_uri_mock  # pylint: disable=W0611
+
+import pandas as pd
+import sklearn.datasets as datasets
+from fastai.tabular import tabular_learner, TabularList
+from fastai.metrics import accuracy
+import mlflow  # noqa
+import mlflow.fastai  # noqa
+from fastai.callbacks import EarlyStoppingCallback, SaveModelCallback
+
+np.random.seed(1337)
+
+LARGE_EPOCHS = 10
```

Comment:
For the follow-up PR containing the autologging functionality, might make sense to pick a smaller number here to help ensure the test runs quickly.